evap/student/templates/student_vote.html

```diff
@@ -299,9 +298,14 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                 button.addEventListener("click", () => {
                     const contributorId = button.dataset.markNoAnswersFor;
                     const voteArea = document.querySelector(`#vote-area-${contributorId}`);
-                    // check "no answer" and uncheck all other options
-                    voteArea.querySelectorAll(".vote-inputs [type=radio][value='6']")
-                        .forEach(radioInput => radioInput.click());
+
+                    voteArea.querySelectorAll(".vote-btn").forEach(el => el.classList.remove("active"));
```

Comment:
I don’t know if it makes a difference, but wouldn’t this be even faster when pre-selecting with `voteArea.querySelectorAll(".vote-btn.active")`?